src/scenes/AcceleratorRouter/RejectedDeliverableMessage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Box, useTheme } from '@mui/material';
+import { Button, Message } from '@terraware/web-components';
+import { ViewProps } from 'src/components/DeliverableView/types';
+import useAcceleratorConsole from 'src/hooks/useAcceleratorConsole';
+import { useLocalization } from 'src/providers/hooks';
+import strings from 'src/strings';
+
+const RejectedDeliverableMessage = ({ deliverable }: ViewProps): JSX.Element => {
+  const { isAcceleratorRoute } = useAcceleratorConsole();
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+
+  const onClickEditFeedback = () => {
+    alert('TODO: Edit rejection feedback');
```

Comment:
@joshbuchea tagged you in a design [comment](https://www.figma.com/file/OlTouFTKgxUPX8ejhY6mMv?node-id=2489:82569&mode=design#719376720). Edit feedback calls the same reject flow.
Captured that here [ticket](https://terraformation.atlassian.net/browse/SW-4949) as a separate effort.